agent/util-scripts/test-bin/scp

```diff
@@ -6,6 +6,10 @@ echo "${0} ${*}" >> $_testlog
 src="${@:$(( ${#} - 1 ))}"
 # Fetch the last argument as the "destination" of the copy.
 dst="${@:$#}"
+echo "$0 $*" >> $_testlog
```

Comment:
Why is this file part of this PR at all? Part of the reason for merging #1412 before this one was to get rid of the need to change this file.

Reply:
This seems like a bad merge.